libs/comments/website/src/lib/comment-editor/comment-editor.tsx

```diff
@@ -408,23 +474,41 @@ export const CommentEditor = ({
 
           {!showInitialModal && (
             <LoginWrapper>
-              <LoginFormContainer
-                afterLoginCallback={handleAfterLoginCallback}
-              />
-
-              <Register>
-                <Button
-                  css={buttonStyles}
-                  onClick={registerRedirect}
+              <ExistingAccountsLoginWrapper>
+                <H1
+                  css={existingAccountsHeadingStyles}
```

Comment:
If you want to use the styling of H4, then use H4. 

As discussed, you should add a comoponent to it which controls the html element. H1 is not the same as h1.

Also needs to be gotten from useWebsiteBuilder as it is an overrideable component.